sqlmesh/core/config/loader.py

```diff
@@ -99,9 +101,15 @@ def load_config_from_paths(
                 )
             non_python_configs.append(load_config_from_yaml(path))
         elif extension == "py":
-            python_config = load_config_from_python_module(
-                config_type, path, config_name=config_name
-            )
+            try:
+                python_config = load_config_from_python_module(
+                    config_type, path, config_name=config_name
+                )
+            except ValidationError as e:
+                raise ConfigError(
+                    validation_error_message(e, f"Invalid project config '{config_name}':")
+                    + "\n\nVerify your config.py."
```

Comment:
```suggestion
                    + "\n\nVerify your config.py and environment variables."
```

Reply:
We don't merge environments with config.py today like we do for config.yaml, so the message would be misleading.  See https://github.com/TobikoData/sqlmesh/issues/4106

Reply:
ahh thanks, that's helpful to know